pkgs/applications/networking/flexget/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, python
+{ lib, python36, python27
```

Comment:
Can't we just use `python3`? Seems like upstream supports 3.7: https://github.com/Flexget/Flexget/commit/69216697d3513132bcfa25b7838caf758929b342

Reply:
it fails to build with python 3.7, which is used when python3 is set.

Reply:
What's the error message? You should report that upstream.

Reply:
Why?

https://flexget.com/Install -> Supported Python versions: 2.7, 3.4, 3.5 and 3.6.

Reply:
That's outdated. As you will see when you look at the commit linked above, 3.7 is supported since 2.17.18.

Reply:
Sorry, maybe it's not. Python 3.7 support migt be work in progress.